### ISSUE TITLE
Magazine demo doesn't work on IE9

### ISSUE THREAD

**Issue by warmth** [blasten/turn.js#64]
What should I do to make the Magazine demo to work on IE9 like the bible demo & your website does?

UPDATE: opening the IE9 console & refreshing make the script to work, very weird.

**Comment by blasten** [comment]
Thanks for let me know. I promise to check it out tomorrow.

**Comment by warmth** [comment]
Thank you pal. I'll appreciate that so much because I need to finish a project for this Friday.

**Comment by warmth** [comment]
Found that many of the problems come because of the console.log on IE and your Magazine demo has one:

console.log('Current view: ', $(this).turn('view'));

any news of the IE7/IE8 support?

**Comment by blasten** [comment]
It seems like console.log in IE9 is only defined when the developer tools are opened.  I will remove it from the samples.

**Comment by warmth** [comment]
That's exactly what I thought... Any news about turn-ie8.js? I won't complain beta testing it on my website... better than not having support at all! LOL!

**Comment by blasten** [comment]
@warmth turnjs-ie8.  will be part of the next version. http://turnjs.com/licenses.pdf

**Comment by vagabondcreative** [comment]
Hello,

I am looking at purchasing and implementing V4 with zoom for a client as it is exampled on this page:

http://turnjs.com/samples/magazine/

However I'm noticing that this example does not work in IE8 which is a priority for my client.

```
Webpage error details

Message: Object doesn't support this property or method
Line: 38
Char: 416
Code: 0
URI: http://turnjs.com/lib/turn.html4.min.js
```

It appears V4 is supposed to support IE8 : http://turnjs.com/features.pdf

Can you please tell me if there is anything that can be done? My client is looking to move on this as soon as possible so I look forward to your response.

Thanks!

**Comment by blasten** [comment]
@vagabondcreative  there must be something missed in the html4 version since zoom.js uses top new features of turn.js. I can take a look at it and solve it ASAP

**Comment by vagabondcreative** [comment]
Thanks @blasten !!!

This client is hoping to make decision within the week. I'll look forward to your response.

Cheers :)

**Comment by blasten** [comment]
@vagabondcreative  The problem was solved.

**Comment by meerkatlookout** [comment]
We recently purchased your library (which is great btw – well done) but I am having trouble running the demos (like "basic" or "magazine") that comes with the zip download in IE8?

Your updated online samples do in fact seem to work fine in IE8 but I cannot get those in the download package to work correctly in IE8. Comes up with a similar error as mentioned earlier in this issue :

```
Webpage error details

Message: Object doesn't support this property or method
Line: 38
Char: 445
Code: 0
URI: XXXX/lib/turn.html4.min.js
```

It’s probably just a matter of updating the zip samples for download but I’d appreciate if you could direct me as to how to get this to work?

**Comment by blasten** [comment]
@meerkatlookout could you send me an email to blasten@gmail.com?

**Comment by blasten** [comment]
@meerkatlookout This problem was fixed in 4.0.7
